File: project 5 - function/chessCreator.py
def print_chessboard(n, m):
    for i in range(n):
        row = ""
        for j in range(m):
            if (i + j) % 2 == 0:
                row += "*"
            else:
                row += "#"
        print(row)

n = int(input("row => "))
m = int(input("col => "))  
print_chessboard(n, m)

File: project 5 - function/php/chessCreator.php
<?php
    if (isset($_GET['n']) && isset($_GET['m'])) {
        $row = (int)$_GET['n'];
        $col = (int)$_GET['m'];

        function print_chessboard($row, $col) {
            for ($i = 0; $i < $row; $i++) {
                $line = '';
                for ($j = 0; $j < $col; $j++) {
                    if (($i + $j) % 2 == 0) {
                        $line .= "*";
                    } else {
                        $line .= "#";
                    }
                }
                echo "<div>$line</div>";
            }
        }

        print_chessboard($row, $col);
    } else {
        echo "Failed: Please provide valid parameters.";
    }